builder/docker/config.go

```diff
@@ -12,23 +12,33 @@ import (
 	"github.com/mitchellh/packer/template/interpolate"
 )
 
+var (
+	ErrArtifactNotUsed     = fmt.Errorf("No instructions given for handling the artifact; expected commit, discard, or export_path")
+	ErrArtifactUseConflict = fmt.Errorf("Cannot specify more than one of commit, discard, and export_path")
+	ErrExportPathNotFile   = fmt.Errorf("export_path must be a file, not a directory")
+	ErrImageNotSpecified   = fmt.Errorf("Image must be specified")
```

Comment:
This style is fine, but I'm not a fun of exposing them (uppercase letter) since there is no API guarantee these will stick around if the config validation changes (i.e. Docker changes).